liota/entities/devices/RAM.py

```diff
@@ -30,4 +30,31 @@
 #  THE POSSIBILITY OF SUCH DAMAGE.                                            #
 # ----------------------------------------------------------------------------#
 
-# TODO: Implement this module
+from abc import ABCMeta, abstractmethod
```

Comment:
Is this example providing any value add? I see that it is just extending the device class as-is. In the current form, it is not even changing the entity_type. I think we should add it only to demonstrate additional functionality.